src/components/Modal/Translate.vue

```diff
@@ -172,20 +191,67 @@ export default {
 		async contentReplace() {
 			this.$emit('replace-content', this.result)
 		},
+		autosize() {
+			this.$refs.input.style.height = 'auto'
+			const height = this.$refs.input.scrollHeight
+			this.$refs.input.style.height = height + 'px'
+			this.$refs.result.style.height = height + 'px'
+		},
+		onInputFocus() {
+			this.disableFromLanguageSelect = false
+			this.autosize()
+		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
 .translate-dialog {
 	margin: 24px;
+
+	.wrapper {
+		display: grid;
+		grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+		grid-column-gap: 16px;
+		margin-top: calc(var(--default-grid-baseline) * 6);
+
+		.language-selector {
+			font-weight: bold;
+			flex-wrap: wrap;
+			gap: var(--default-grid-baseline);
+		}
+
+		.col {
+			grid-row: 1/2;
+		}
+	}
 }
 
 textarea {
 	display: block;
 	width: 100%;
 	margin-bottom: 12px;
+	height: auto;
+	resize: none;
+	box-sizing: border-box;
+	overflow: hidden;
+	max-height: 60vh;
+
+	&.result {
+		overflow: scroll;
```

Comment:
Why just for the result? Let's remove the `.result` class and set `overflow-y: auto` for both, no?